projects/components/src/multi-select/multi-select.component.ts

```diff
@@ -362,6 +365,30 @@ export class MultiSelectComponent<V> implements ControlValueAccessor, AfterConte
     this.propagateControlValueChange?.(value);
     this.propagateControlValueChangeOnTouch?.(value);
   }
+
+  private reorderSelectedItemsFirst(
+    filteredOptions: SelectOptionComponent<V>[],
+    selected?: V[]
+  ): SelectOptionComponent<V>[] {
+    if (isEmpty(selected)) {
+      return filteredOptions;
+    }
+    const filteredOptionsPartitions = compact(
```

Comment:
compact is not needed here since there won't be any falsey values in the array.

Reply:
done